lib/node/runner/api/transaction_test.go

```diff
@@ -49,6 +49,7 @@ func TestGetTransactionByHashHandler(t *testing.T) {
 		json.Unmarshal(readByte, &recv)
 
 		require.Equal(t, bt.Hash, recv["hash"], "hash is not same")
+		require.Equal(t, bt.Block, recv["block"], "block is not same")
```

Comment:
I know the mistake is already there, but the proper message would be "block is not the same".
However, since `Equal` already shows that message with a diff, I think that message is redundant.

Reply:
@Geod24 Ok, I revised the message.